### PR TITLE
Random port mapping and warnings

### PR DESCRIPTION
**Pull request by TeckniX**
There were some simple warnings associated with the objects passed that were fixed - I also added a patch to the 'Random port issue' https://github.com/kitematic/kitematic/issues/530 

Port mapping settings view will also be updated shortly, as it's referencing the wrong object.

### CODE REVIEW

File: src/components/ContainerDetailsSubheader.react.js [hunk 1]
```diff
@@ -83,7 +83,8 @@ var ContainerDetailsSubheader = React.createClass({
   handleRestart: function () {
     if (!this.disableRestart()) {
       metrics.track('Restarted Container');
-      dockerUtil.restart(this.props.container.Name);
+      //dockerUtil.restart(this.props.container.Name);
```
Comment (jeffdm) [inline src/components/ContainerDetailsSubheader.react.js]: Remove comment if line below works

**Comment by mchiang0610** [comment]
Thanks! Checking it out.

**Comment by jeffdm** [comment]
@TeckniX thank you!!

One small comment, then LGTM

**Comment by TeckniX** [comment]
DUH! forgot to remove some old comments - The restart is now streamlined through the ContainerAction, instead of the direct call - let me fix it

**Comment by jeffdm** [comment]
@TeckniX awesome! LGTM! Merging.

**Pull request merged**